packages/babel-plugin/__tests__/transform-import-export-test.js

```diff
@@ -0,0 +1,527 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+'use strict';
+
+jest.autoMockOff();
+
+const { transformSync } = require('@babel/core');
+const stylexPlugin = require('../src/index');
+
+function transform(source, opts = {}) {
+  const { code, metadata } = transformSync(source, {
+    filename: opts.filename || '/stylex/packages/vars.stylex.js',
+    parserOpts: {
+      flow: 'all',
+    },
+    plugins: [
+      [
+        stylexPlugin,
+        {
+          unstable_moduleResolution: {
+            rootDir: '/stylex/package/',
+            type: 'commonJS',
+          },
+          ...opts,
+        },
+      ],
+    ],
+  });
+
+  return { code, metadata };
+}
+
+/**
+ * Fixture factory
+ *
+ * This is used to create a consistent fixture across all the tests for
+ * different ways of importing StyleX. The fixture uses all of the
+ * StyleX exports to make sure they are transformed as expected.
+ */
+
+const defaultImportSource = {
+  imprt: '* as stylex',
```

Comment:
typo?

Reply:
I think this is intentional. I think adding an `_` is preferrable than dropping the `o`.